### ISSUE TITLE
Document -S (--string) in language reference (and maybe tutorial, too).

### ISSUE THREAD

**Issue by gjolund** [google/jsonnet#852]
Not much else to say.

```sh
$ jsonnet --version && cat test.jsonnet && echo  && jsonnet test.jsonnet 
Jsonnet commandline interpreter v0.16.0
local wrapper = [
  {
    some_prop: true
  }
];

std.manifestYamlDoc(wrapper)
"- \"some_prop\": true"
```

**Comment by seh** [comment]
Try passing the `--string` flag to the _jsonnet_ tool.

**Comment by sbarzowski** [comment]
Yep. Jsonnet formats the output as JSON by default. The function manifestYamlDoc returns a string, so the output is formatted as JSON string which contains your YAML. When you pass `-S`/`--string`, the output string is printed verbatim.

**Comment by austinrivas** [comment]
Do you mind pointing me to where in the documentation it says this?

I have been going off this and it did not mention any of this https://jsonnet.org/ref/stdlib.html

**Comment by sbarzowski** [comment]
![image](https://user-images.githubusercontent.com/683828/96420518-e6b1be00-11f5-11eb-8e7c-5ce90eadb0bc.png)

That said, we should probably document `-S` option in reference and/or tutorial.

**Comment by austinrivas** [comment]
Thanks for the response. A line or two of options docs would have saved me an hour of debugging there

**Comment by sbarzowski** [comment]
I think we can still document it better. This is a pretty basic feature and should be mentioned in the tutorial. Reopening.
